Source/WebCore/Modules/applepay-ams-ui/ApplePayAMSUIPaymentHandler.cpp

```diff
@@ -128,11 +128,11 @@ ExceptionOr<void> ApplePayAMSUIPaymentHandler::convertData(Document& document, J
     return { };
 }
 
-ExceptionOr<void> ApplePayAMSUIPaymentHandler::show(Document& document)
+ExceptionOr<void> ApplePayAMSUIPaymentHandler::show(Document&)
```

Comment:
Remove this parameter altogether?

Reply:
This is overriding a virtual function; the other function that overrides it still requires the parameter.